### PR TITLE
move credentials to Secret

### PR DESCRIPTION
**Pull request by tamcore**
This touches on my comment in https://github.com/traccar/traccar-helm/issues/27#issuecomment-1848965936, as ConfigMaps are not intended to hold confidential data.

### CODE REVIEW

**Comment by tananaev** [comment]
@filippolmt would you mind taking a look.

File: charts/traccar/templates/configmap.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ data:
     <!DOCTYPE properties SYSTEM 'http://java.sun.com/dtd/properties.dtd'>
     <properties>
         <entry key='config.default'>./conf/default.xml</entry>
+        <entry key='config.useEnvironmentVariables'>true</entry>
```
Comment (filippolmt) [inline charts/traccar/templates/configmap.yaml:16]: @tananaev by adding this configuration in the xml file, is it possible to use environment variables?

**Comment by filippolmt** [comment]
As an improvement, I would add the possibility of using an existing secrets in the cluster (of course documenting how to do this) and adding a functionality test.

The fact remains that this PR is fine, you could opt to do it at a later date.

**Review by filippolmt — APPROVED** [review]

File: charts/traccar/templates/configmap.yaml (hunk 1 above)
Comment (tananaev) [reply charts/traccar/templates/configmap.yaml:16]: Yes.

**Comment by filippolmt** [comment]
Thank you, I had completely missed it. :)

**Comment by filippolmt** [comment]
@tananaev we can merge the PR

**Comment by tananaev** [comment]
Thanks

**Pull request merged**